scripts/mdtopdf.sh

```diff
@@ -0,0 +1,122 @@
+     # Remove INPUT_FOLDER from filename
+     local PDF_FILE_NAME="${SOURCE_FILE#$INPUT_FOLDER}"
+     # Remove leading ./ or / from filename, as find will output files with leading ./ or /
+     local PDF_FILE_NAME="${PDF_FILE_NAME#*/}"
+     # Replace all "/" characters in filename to "-" and append .pdf
+     local PDF_FILE_NAME="${FILE_PATH//\//-}"
+     # Change file extension from .html to .pdf
+     local PDF_FILE_NAME="${PDF_FILE_NAME/%.md/.pdf}"
+     echo $PDF_FILE_NAME "file name"
+     # For example if SOURCE_FILE=./build/1.10/cli/dcos-marathon-group-scale-index.html
+     # PDF_FILE_NAME will be 1.10-cli-dcos-marathon-group-scale-index.html.p
+     # Make the Destination directory
+     mkdir -p "${PDF_DEST_DIR}"
+
+     # There is an index.md whithing every file
+     FILE_NAME="index.md"
+     TEMP_FILE=$(mktemp)
+
+    # echo "find ${INPUT_FOLDER}/${FILE_PATH} -type d -depth"
+
+    # We find the index.md per folder so the final pdf is organised per folder not natively recursive
+    while IFS= read -r SOURCE_FOLDERS
+      do
+        # Target all the folder names
+        local d="$SOURCE_FOLDERS"
+        # Target all the files whithin the foler by the same name
+        NEW_FILE="${d}/${FILE_NAME}"
+
+        if [ -f "${NEW_FILE}" ]
+        then
+          # Create temporary file with all md content to send to pandoc // this avoids very long urls & long strings (Pandoc has a string limit)
+          TEMP_FILES="${TEMP_FILES} ${TEMP_FILE}"
+          echo "" >> "${TEMP_FILE}"
+          cat "${NEW_FILE}" >> "${TEMP_FILE}"
+        fi
+      # Find recursively all the directories whithin a folder
+      done < <(find "${INPUT_FOLDER}"/"${FILE_PATH}" -type d -depth)
+
+     # Fix for absolute paths images urls
+     sed -i 's,\([:[:space:](]\)/\([-0-9A-Za-z/_.]*\.png\),\1\2,g;s,\([:[:space:](]\)/\([-0-9A-Za-z/_.]*\.jpg\),\1\2,g;s,\([:[:space:](]\)/\([-0-9A-Za-z/_.]*\.jpeg\),\1\2,g;s,\([:[:space:](]\)/\([-0-9A-Za-z/_.]*\.gif\),\1\2,g;s,\([:[:space:](]\)/\([-0-9A-Za-z/_.]*\.svg\),..\1\2,g' "${TEMP_FILE}"
+     sed -i 's,@,at,g' "${TEMP_FILE}"
+
+     # Unicode characters to encode into UTF8.
+     CHARS=$(python -c 'print u"\u2060\u0080\u0099\u009C\u009d\u0098\u0094\u0082\u00a6\u0089\u00a4\u00a5\u0093\u2019\u2018\u201C\u201D\u25CF".encode("utf8" )')
+     sed -i 's/['"$CHARS"']//g' "${TEMP_FILE}"
+
+     # We cut all long strings at 180 characters.
+     sed -i -r 's/.{180}/&\n/g' "${TEMP_FILE}"
+
+     sed -i -r 's,\n,\\n,g' "${TEMP_FILE}"
+     sed -i -r 's,_,\_,g' "${TEMP_FILE}"
+
+     #Fix for all local links to not mislead Pandoc into grabbing other files when there is nothing to look for
+     # issue reported in Pandoc : https://github.com/jgm/pandoc/issues/3619
+     sed -i -r 's,]:,]\\:,g' "${TEMP_FILE}"
+     sed -i -r 's,\/#,,g' "${TEMP_FILE}"
+
+    # Set name for last folder
+    if [ -z "${PDF_FILE_NAME}" ]
+    then
+      "${PDF_FILE_NAME}"="MesosphereDCOS"
```

Comment:
i think this should be 

`PDF_FILE_NAME="MesosphereDCOS"` correct?

Reply:
Correct, I'll do that change today.

Reply:
This is also fixed as per the second to last commit.